bundle/Handler/FieldType/RichText.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\OpenGraphBundle\Handler\FieldType;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\RichText\Value;
+use Netgen\Bundle\OpenGraphBundle\Exception\FieldEmptyException;
+
+class RichText extends XmlText
```

Comment:
If the signature of the value is the same, why not just add the rich text value check in the original `supports` method and just tag the original implementation with a new tag in `services.yml`?

Also, I plan to mark the all classes final before the next release, so that is one more incentive not to extend :)

Reply:
Yes it' possible. But why close your code ?

Reply:
Not sure what you mean.

In eZ kernel 7 rich text was included in eZ kernel. In v8, it was taken out to a separate package (https://github.com/ezsystems/ezplatform-richtext).

Additionally, there's a separate ezxmltext package which can be used on both v7 and v8.

In any case, if you wish to use old richtext namespace, this PR needs to go to v1 branch of this bundle.

As for `final`,  its simply because of best practices and less maintenance costs.